### ISSUE TITLE
Botania Export Mana

### ISSUE THREAD

**Issue by HamzaYazaw** [AllTheMods/ATM-4#107]
Hello everyone, 
I was just messing around with Botania, and the Elven Mana Spreader, is not exporting to the mana pool

Any ideas? 
![Base Profile Screenshot 2020 01 03 - 14 30 10 11](https://user-images.githubusercontent.com/47602927/71723570-d2066400-2e35-11ea-9582-fca21b58d31d.png)

**Comment by LezChap** [comment]
Some questions:
Is it just that spreader, or all spreaders that are failing to export?
How many of the Endoflames are connected to each spreader?
If you stop feeding the Endoflames, does it empty out when they stop producing mana?

I'm suspecting that there's more mana getting inputted into the spreader from the Endoflames while they're consuming fuel than that spreader can output to the Mana Pool.

**Comment by HamzaYazaw** [comment]
@LezChap 
Hello, thank you for responding 
So I just did a little test regarding your questions

It's all 3 "Elven Spreader" while the normal "Mana Spreader" works perfectly 
I have connected 12 on both sides so the total is 24 
I have tried stopping feeding the spreader, apparently it exports from it but there is no mana going in the pool

You can see the attached screenshots on what I mean on the last point

![Base Profile Screenshot 2020 01 03 - 17 49 22 55](https://user-images.githubusercontent.com/47602927/71733101-6fbb5c80-2e51-11ea-877b-f80cc3c98a59.png)
![Base Profile Screenshot 2020 01 03 - 17 49 24 57](https://user-images.githubusercontent.com/47602927/71733102-6fbb5c80-2e51-11ea-8650-cb09e157f916.png)

**Comment by LezChap** [comment]
There's no numbers visible to the player for Mana in Botania.  That's intentional.  But it's still a program, so there are numbers.  A mana spreader, afaik, must have a minimum amount before a burst will fire.  Let's say those spreaders hold 1000 units of mana (I think this is the actual number, but I'd have to look it up and confirm).  I believe a burst size for those spreaders is 240 mana.  If the mana in the spreader is <240 mana, it doesn't have enough to output.  If the numbers I remember (1000 and 240) are accurate, that last screenshot you may have less than 240 mana in the spreader, and thus it's unable to fire.  It did fire between the second to last and last screenshots you posted...so it's outputting.  It just appears that it's filling faster then it can output in the OP, and doesn't have enough mana to make a burst in the last.

**Comment by HamzaYazaw** [comment]
@LezChap 
Yes I have read that it needs above quarter of the mana bar basically. Which will only fire above 240 

- I only provided those screenshots to show that they are actually firing at some point but the mana is not going anywhere, even tried to make it input less, the mana isn't filling within the Mana Pool

**Comment by LezChap** [comment]
What makes you sure that it's not going into the mana pool?  Mana Pools can store like 1,000,000 mana.  A pulse of 240 doesn't make a visual change on the bar of the Mana Pool.

**Comment by HamzaYazaw** [comment]
@LezChap 
by using the Wand where you can actually check how much it did.
Because using 1 Mana Spreader (The normal one) does output and YES, you  can visually see that inside the mana pool.

**Comment by RViper80** [comment]
I've had issues in this modpack with visual glitches -- after logging on, for example, I would see only half of any beds, some storage drawers look empty, some botania blocks aren't there until you put the crosshair over them, and look wrong when they do.  But I've also added 'Create' so that might be causing some of that.

I have had an issue though with a couple spreaders not hitting the pool after loading the world until I re-bound them.  I did that 2-3 times, and after that I used the function mode of the Wand of the Forest to aim it manually.  Once I did that, it has stayed aligned ever since.

Not sure if that is your issue or not, but it's something I've ran across.  Also as someone mentioned above, the elven spreaders will fire slowly with only about six endoflames on each one.  I've got 15 endoflames on one of my livingwood spreaders (tight against a pool, with a potency lens) and it's keeping up just fine to them.
